src/pages/roadmap.md

```diff
@@ -6,42 +6,43 @@ hide_table_of_contents: true
 
 Timelines are estimates and subject to change. For information about delivered features and fixes, see the [Changelog](/docs/changelog).
 
-## Release cycle
+## 2023 milestones
 
-We typically follow this release cycle:
+### Q4
 
-- Alpha: Super early peek into the feature being built. Not ready for use in production apps.
-- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps.
-- Production: Ready for use in production apps.
+- Allow/denylist user preferences available in all SDKs
 
-## Features roadmap
+## 2024 milestones
 
-### 2023 Q3
+### Q1
 
-- Onchain transaction reference content type
-- React SDK in Production
-- React Native SDK in Production (currently in Beta)
-- Android SDK in Production (currently in Beta)
-- XMTP v3 in Alpha
+- [Group chat via Messaging Layer Security (MLS)](https://github.com/xmtp/roadmap/milestone/1) in alpha
+- [Contact directory via did:ethr](https://github.com/xmtp/roadmap/milestone/5) in alpha
+- [XMTP testnet](https://github.com/xmtp/roadmap/milestone/6) in alpha
 
-### 2023 Q4
+### Q2
 
-- XMTP v3 in Beta
+- [Group chat via MLS](https://github.com/xmtp/roadmap/milestone/2) in production
+- [XMTP testnet](https://github.com/xmtp/roadmap/milestone/6) in production
+- [1:1 chats via MLS](https://github.com/xmtp/roadmap/milestone/3) in production
 
-### 2024 Q1
+### Q3
 
-- XMTP v3 in Production
-- Support for smart contract wallets
-- Portable preferences in Alpha - blocklists, allowlists, etc.
+- [XMTP fully using MLS](https://github.com/xmtp/roadmap/milestone/4) in production
```

Comment:
Not sure if I captured this one correctly - tried to get a little bit more specific than "MLS feature complete." What does this mean - that anything that can use MLS - will be using MLS?

Reply:
The Q1 milestone will use MLS only for group chat; the goal for the Q3 milestone is to base 1:1 chat on MLS, replacing our existing double-ratchet code.

I can't think of a reason why the above would make 'fully using' inaccurate, but maybe some version of the above is a clarity win?

Reply:
ok, looking at the page (rather than the diffs), my suggestion would be to remove the "fully using" bullet point from Q3, and move the "1:1 chats via MLS" bullet point in.

Reply:
You got it, @bwcDvorak - thank you! 🖖